docker-compose.yml

```diff
@@ -14,9 +14,8 @@ services:
     image: "srcd/gitbase:v0.18.0"
     environment:
       BBLFSH_ENDPOINT: bblfsh:9432
-      GITBASE_UNSTABLE_SQUASH_ENABLE: "true"
     volumes:
-      - ${GITBASEPG_REPOS_FOLDER}:/opt/repos
+      - ${GITBASEPG_REPOS_FOLDER-./repos}:/opt/repos
```

Comment:
I'm not sure about this change. The docs mention `$HOME/repos`, not `./repos`.
Also, not having a default value forces the first-time user to pay attention and read the docs. Preparing a directory with a few repos is mandatory, and starting the containers with no errors but also no data can be more confusing.

WDYT @dpordomingo?

Reply:
I agree with @carlosms 
It's a good idea to avoid starting the playground without repositories in the DB, or at least without forcing the user to explicitly define a place.

about using `./repos` or `$HOME/repos`, I'm ok with both, but it should be consistent with docs.